tests/python_api/audio_unit_test.py

```diff
@@ -0,0 +1,178 @@
+# Copyright (c) 2024 Advanced Micro Devices, Inc. All rights reserved.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+from amd.rocal.pipeline import Pipeline
+from amd.rocal.pipeline import pipeline_def
+from amd.rocal.plugin.pytorch import ROCALAudioIterator
+import amd.rocal.fn as fn
+import random
+import os
+import sys
+import matplotlib.pyplot as plt
+import torch
+import numpy as np
+from parse_config import parse_args
+
+np.set_printoptions(threshold=1000, edgeitems=10000)
```

Comment:
Change edgeitems to edge_items

Reply:
this arg is built in of the numpy, we will not be able to change this

Reply:
There are 2 set_printoptions. Check if both are required.